cis/system/hsp-cis-1.1.7-etcd-pod-block.yaml

```diff
@@ -3,8 +3,8 @@ kind: KubeArmorHostPolicy
 metadata:
   name: hsp-cis-1-1-7-etcd-pod-block
 spec:
-  tags: ["CIS", "1.1.7", "K8s"]
-  message: "This policy will block kube-scheduler.yaml access"
+  tags: ["CIS", "1.1.7", "k8s"]
+  message: "etcd pod specification file permissions are set to 644"
```

Comment:
Can you change the Message here, since it's blocked craft it more reasonable
 - [ ] Policy message is not clear